src/ingest_validation_tools/directory-schemas/codex.yaml

```
@@ -1,65 +1,72 @@
-# A dataset consists of a single CODEX experimental run, and is
-# collected by the Akoya software into a single directory.
-#
-# directory beginning with drv_ contains analyzed files produced.
-# There are a few files in this directory that are used for the HuBMAP
-# data processing pipelines (channelNames.txt, experiment.json, segmentation.json)
-#
-# directory beginning with src_ contains the raw (unprocessed by Akoya) files.
+# A dataset consists of a single CODEX experimental run
 
 -
   pattern: '[^/]*NAV[^/]*\.tif'
   description: 'Navigational Image showing Region of Interest (Keyance Microscope only)'
   required: False
 -
   example: 'summary.pdf'
-  pattern: '.+\.pdf'
+  pattern: '[^/].*\.pdf'
```

```suggestion
  pattern: '[^/]*\.pdf'
```
- Is your intention that the pdf be at the top level?
- We know nothing else about the file name?
- Are we expecting it from one lab but not the other? Might be useful to explain in a comment.

---

@mccalluc 

* Correct, a single PDF at the top level
* No. Originally named `summary.pdf` but you suggested some time ago to change it to a regex.
* Just from UFL at the moment.